xlml/apis/test_config.py

```diff
@@ -480,10 +476,45 @@ class JSonnetGpuTest(TestConfig[Gpu]):
   """
 
   test_name: str
-  test_command: List[str]
   entrypoint_script: List[str]
+  test_command: List[str]
   docker_image: str
+  accelerator: Gpu
```

Comment:
`accelerator`'s type is derived from the parent type's parameter

Reply:
Do you mean we don't need to mentioned `accelerator: Gpu` here because it is deticed from `TestConfig[Gpu]`?